community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/variables.tf

```diff
@@ -85,7 +85,7 @@ variable "preserve_tpu" {
 }
 
 variable "zone" {
-  description = "Zone in which to create compute VMs. Additional zones in the same region can be specified in var.zones."
+  description = "Zone in which to create compute VMs. Zone-finding is not currently available."
```

Comment:
nit: I would instead say: "Zone in which to create compute VMs. TPU partitions can only specify a single zone."

Zone-finding is not really an official term.

Reply:
Makes sense, thanks for the tip.